revapi-java/src/site/asciidoc/index.adoc

```diff
@@ -58,7 +58,2479 @@ class formally part of the API even though it comes from a dependency. The `Mode
+| `methodName` | the name of the method added
+| `oldModifiers` | the modifiers on the method in the old version
+| `newModifiers` | the modifiers on the method in the new version
+| `elementKind` | See <<Match Parameters>> for possible values
+|====
+
+==== Method Now Abstract
+[cols="35s,<65d"]
+|=============
+| Code              | +java.method.nowAbstract+
+| Binary severity   | breaking
+| Source severity   | breaking
+| Semantic severity | NA
+|=============
+
+If a method becomes abstract, all the inheriting classes will have to implement it even though they didn't have
+to before.
+
+[options=header]
+|====
+2+| Match parameters
+| `package` | the package of the class
+| `classSimpleName` | the simple name of the class
+| `methodName` | the name of the method added
+| `oldModifiers` | the modifiers on the method in the old version
+| `newModifiers` | the modifiers on the method in the new version
+| `elementKind` | See <<Match Parameters>> for possible values
+|====
+
+==== Method No Longer Abstract
+[cols="35s,<65d"]
+|=============
+| Code              | +java.method.noLongerAbstract+
+| Binary severity   | non breaking
+| Source severity   | non breaking
+| Semantic severity | NA
+|=============
+
+This is a compatible change reported for the completeness' sake.
+
+[options=header]
+|====
+2+| Match parameters
+| `package` | the package of the class
+| `classSimpleName` | the simple name of the class
+| `methodName` | the name of the method added
+| `oldModifiers` | the modifiers on the method in the old version
+| `newModifiers` | the modifiers on the method in the new version
+| `elementKind` | See <<Match Parameters>> for possible values
+|====
+
+==== Method Visibility Increased
+[cols="35s,<65d"]
+|=============
+| Code              | +java.method.visibilityIncreased+
+| Binary severity   | equivalent
+| Source severity   | equivalent
+| Semantic severity | NA
+|=============
+
+No API breakage, reported for completeness' sake.
+
+[options=header]
+|====
+2+| Match parameters
+| `package` | the package of the class
+| `classSimpleName` | the simple name of the class
+| `methodName` | the name of the method added
+| `oldVisibility` | The visibility of the type as it was in the old version.
+| `newVisibility` | The visibility of the type in the new version.
+| `elementKind` | See <<Match Parameters>> for possible values
+|====
+
+==== Method Visibility Reduced
+[cols="35s,<65d"]
+|=============
+| Code              | +java.method.visibilityIncreased+
+| Binary severity   | breaking
+| Source severity   | breaking
+| Semantic severity | NA
+|=============
+
+A method might no longer be visible to code that used to call it. This is a breaking change.
+
+[options=header]
+|====
+2+| Match parameters
+| `package` | the package of the class
+| `classSimpleName` | the simple name of the class
+| `methodName` | the name of the method added
+| `oldVisibility` | The visibility of the type as it was in the old version.
```

Comment:
This link should not be mentioned (and shouldn't have been before your changes either 😉  ).